### PR TITLE
[New Rule] T1543.003 - Unsigned DLL Loaded by Svchost

### PR DESCRIPTION
**Pull request by Samirbous**
image

Compatible with Endpoint 8.4+ only

### CODE REVIEW

File: rules/windows/persistence_service_dll_unsigned.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2023/01/17"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
```
Comment (imays11) [inline rules/windows/persistence_service_dll_unsigned.toml:5]: What's the reason for min_stack : 8.4 ?

Comment (Samirbous) [reply rules/windows/persistence_service_dll_unsigned.toml:5]: `dll.Ext.relative_file_creation_time` is only available on Elastic Endpoint 8.4+

Comment (imays11) [reply rules/windows/persistence_service_dll_unsigned.toml:5]: can you change the ```min_stack_comments``` to reflect that?

File: detection_rules/etc/non-ecs-schema.json [hunk 2]
```diff
@@ -67,7 +67,12 @@
     "process.Ext.effective_parent.executable": "keyword",
     "file.Ext.header_bytes": "keyword", 
     "file.Ext.entropy": "long",
-    "file.size": "long"
+    "file.size": "long",
+    "dll.Ext.relative_file_creation_time": "double", 
+    "dll.Ext.relative_file_name_modify_time": "double" ,
+    "process.Ext.relative_file_name_modify_time": "double",
+    "process.Ext.relative_file_creation_time": "double"
+
```
Comment (w0rk3r) [inline detection_rules/etc/non-ecs-schema.json:75]:
```suggestion
```

File: rules/windows/persistence_service_dll_unsigned.toml (hunk 1 above)
Comment (w0rk3r) [inline rules/windows/persistence_service_dll_unsigned.toml:5]:
```suggestion
min_stack_comments = "This rule uses the dll.Ext.relative_file_creation_time field, introduced in the 8.4 release."
```

File: rules/windows/persistence_service_dll_unsigned.toml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+min_stack_version = "8.4.0"
+updated_date = "2023/01/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an unsigned library that was created last 5 minutes and subsequently loaded by a shared windows service (svchost). 
+Adversaries may use this technique to maintain persistence or run with System privileges.
```
Comment (w0rk3r) [inline rules/windows/persistence_service_dll_unsigned.toml:13]:
```suggestion
Identifies an unsigned library created in the last 5 minutes and subsequently loaded by a shared windows service
(svchost). Adversaries may use this technique to maintain persistence or run with System privileges.
```

**Review by w0rk3r — APPROVED** [review]
Nice one

File: rules/windows/persistence_service_dll_unsigned.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/persistence_service_dll_unsigned.toml]: done

**Review by terrancedejesus — APPROVED** [review]

**Pull request merged**